### DISCUSSION TITLE
Next.js pages router with vanilla-extract

### DISCUSSION THREAD

**Discussion by codemage8** [mantinedev/mantine#5108] [Q&A]
When I use `vanilla-extract` with Next.js 13 pages router and try to use `vars` created by `themeToVars` in `css.ts` files, 
It fails with `document is undefined`.

![screenshot](https://github.com/mantinedev/mantine/assets/126920981/962997ee-2f7e-4d87-a817-1fe657cbd561)

How can I solve this problem?

**Comment by rtivital** [comment]
It is a known Vanilla extract issue, there is no workaround until it is fixed on their side
